### DISCUSSION TITLE
[질문]retrofit 사용에서 response 에 아무것도 없을때의 질문

### DISCUSSION THREAD

**Discussion by ChaMinXD** [TEAM-DAME/Dame-Android#85] [2. Q&A] [answered]
![image](https://user-images.githubusercontent.com/96709529/202552032-4d3c0c8e-d576-4c83-af14-bd61fd0eb5d4.png)

![image](https://user-images.githubusercontent.com/96709529/202552245-58d18f26-fd88-4fb6-899d-c3160b2aa8e0.png)

response 에 반환값이 없는경우에는 위의 사진 코드를 어떤식으로 해야할까요 ? ?

**Comment by onseok** [comment, marked as answer]
해당 부분은 @kangCoder 와 논의를 해봐야 할 것 같습니다!
